examples/examples/z_pub.rs

```diff
@@ -16,6 +16,7 @@ use clap::Parser;
 use std::time::Duration;
 use zenoh::config::Config;
 use zenoh::prelude::r#async::*;
+use zenoh::sample_builder::SampleBuilderTrait;
```

Comment:
Similar. Trait should be part of the prelude.

Reply:
done